graphblas/operator.py

```diff
@@ -1110,6 +1120,68 @@ def __reduce__(self):
     __call__ = TypedBuiltinUnaryOp.__call__
 
 
+class IndexUnaryOp(OpBase):
+    __slots__ = ("is_positional",)
```

Comment:
Small nit: no need to have this as a tuple (same for `__slots__` of `SelectOp`).